### DISCUSSION TITLE
Installation Problem

### DISCUSSION THREAD

**Discussion by sjafri2** [CrowdStrike/falconpy#69] [Q&A] [answered]
**Describe the bug**
During installation I am getting the below error.
"Could not find a version that satisfies the requirement crowdstrike-falconpy (from versions: )
No matching distribution found for crowdstrike-falconpy"

**To Reproduce**
pip install crowdstrike-falconpy

**Expected behavior**
Installation with out any issue.

**Environment (please complete the following information):**
 - OS: Ubuntu 18.04.5 LTS
 - Python: pip 9.0.1 from /usr/lib/python2.7/dist-packages (python 2.7)

**Additional context**
None

**Comment by jshcodes** [comment, marked as answer]
Due to dependencies, FalconPy only supports Python 3.6+. (Although it does also run on Python 3.5, we stopped unit testing for it when GitHub dropped support for 3.5 runners.)

Have you tried pip3?

```shell
sudo apt install python3-pip
```
and then
```shell
pip3 install crowdstrike-falconpy
```

**Comment by jshcodes** [comment]
Next PR (#67) will contain an update for the README so that the difference between the two PIP calls is more obvious. :-)

**Comment by sjafri2** [comment]
Good morning,
The pip3 command did the magic.
thanks for the help.
